src/components/expression/expression.tsx

```diff
@@ -66,6 +66,10 @@ export type EuiExpressionProps = CommonProps & {
    * Turns the component into a button and adds an editable style border at the bottom
    */
   onClick?: MouseEventHandler<HTMLButtonElement>;
+  /**
+   * Truncates text
+   */
+  truncate?: boolean;
```

Comment:
I'm thinking we may want to make this more scalable by turning it into an [enum](https://github.com/elastic/eui/blob/master/wiki/component-design.md#enums-and-maps). Then it's more explicit to the consumer how the text will behave. Something like:

```suggestion
  /**
   * Truncates text
   */
  textWrap: 'break-word' | 'truncate';
```

Then the default would be `'break-word'`. Which also means moving the `@include euiTextBreakWord` mixin to a specific modifier class.